bofire/data_models/surrogates/mixed_tanimoto_gp.py

```diff
@@ -0,0 +1,50 @@
+from typing import Literal
+
+from pydantic import Field, validator
+
+# from bofire.data_models.enum import MolecularEncodingEnum
+from bofire.data_models.kernels.api import (
+    AnyCategoricalKernal,
+    AnyContinuousKernel,
+    AnyMolecularKernel,
+    HammondDistanceKernel,
+    MaternKernel,
+    TanimotoKernel,
+)
+from bofire.data_models.molfeatures.api import (
+    Fingerprints,
+    FingerprintsFragments,
+    Fragments,
+)
+from bofire.data_models.surrogates.scaler import ScalerEnum
+from bofire.data_models.surrogates.trainable_botorch import TrainableBotorchSurrogate
+
+
+class MixedTanimotoGPSurrogate(TrainableBotorchSurrogate):
+    type: Literal["MixedTanimotoGPSurrogate"] = "MixedTanimotoGPSurrogate"
+
+    continuous_kernel: AnyContinuousKernel = Field(
```

Comment:
We should set a default prior on the lengthscale of the Matern Kernel.

Reply:
And what is about the noise prior?

Reply:
I have added the default prior on the lengthscale of the Matern Kernel and also the noise prior for the MixedTanimotoGP.